src/components/TopicLoader/TopicLoader.js

```diff
@@ -70,6 +70,10 @@ class TopicLoader extends Component {
   onClick(features, layer, event) {
     const { dispatchSetClickedFeatureInfo } = this.props;
 
+    if (!features.length) {
```

Comment:
please remove this check. an empty array indicates that no features are clicked.